src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray`1+Builder.cs

```diff
@@ -420,14 +409,7 @@ public bool Contains(T item)
             /// </summary>
             public T[] ToArray()
             {
-                var tmp = new T[this.Count];
-                var elements = _elements;
-                for (int i = 0; i < tmp.Length; i++)
-                {
-                    tmp[i] = elements[i];
-                }
-
-                return tmp;
+                return (T[])_elements.Clone();
```

Comment:
This is not a safe/equivalent change.  _elements.Length may be larger than Count, with unfilled indices at the end.